README.md

```diff
@@ -20,9 +20,15 @@ python -m ipykernel install --user --name bmorph
 
 ## Usage
 
+
 ### Notebook
 This will be explored in a later release of `bmorph`
 
+**In Progress:**
```

Comment:
Users shouldn't really look in the `docs` directory themselves, but will just go to the documentation website. I would just remove this for now.